app/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
Info: why is the module resolution stategy changed? What is the transformation pipeline between Next, the Typescript compiler and the tool processing Tailwind (which?) ?